test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua

```diff
@@ -0,0 +1,57 @@
+local t = require('luatest')
+local cluster = require('luatest.replica_set')
```

Comment:
Nit: since the module was renamed to `replica_set`, let's call it replica_set in new tests.

Reply:
Fixed.